pkgs/servers/sks/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromBitbucket, ocaml, zlib, db48, perl }:
+
+stdenv.mkDerivation rec {
+  name = "sks-${version}";
+  version = "1.1.6";
+
+  src = fetchFromBitbucket {
+    owner = "skskeyserver";
+    repo = "sks-keyserver";
+    rev = "${version}";
+    sha256 = "00q5ma5rvl10rkc6cdw8d69bddgrmvy0ckqj3hbisy65l4idj2zm";
+  };
+
+  buildInputs = [ ocaml zlib db48 perl ];
+
+  makeFlags = [ "PREFIX=$(out)" "MANDIR=$(out)/share/man" ];
+  preConfigure = ''
+    cp Makefile.local.unused Makefile.local
+    sed -i \
+      -e "s:^LIBDB=.*$:LIBDB=-ldb-4.8:g" \
+      Makefile.local
+  '';
+
+  preBuild = "make dep";
+
+  doCheck = true;
+  checkPhase = "./sks unit_test";
+
+  postInstall = ''
+    mkdir --parents $out/var/lib/sks
+    rm --recursive ./sampleConfig/debian
+    cp ./sampleConfig/* $out/var/lib/sks
+    mkdir $out/var/lib/sks/web
+    cp --recursive ./sampleWeb/* $out/var/lib/sks/web
```

Comment:
Is sks looking for those files in that path? If not, `$out/share` would be a better place to put these files.